homeassistant/components/co2signal/manifest.json

```diff
@@ -1,9 +1,9 @@
 {
   "domain": "co2signal",
-  "name": "CO2 Signal",
+  "name": "Electricity Maps",
   "codeowners": [],
   "config_flow": true,
-  "documentation": "https://www.home-assistant.io/integrations/co2signal",
+  "documentation": "https://www.home-assistant.io/integrations/electricity_maps",
```

Comment:
The domain is not changing.

Reply:
```suggestion
  "documentation": "https://www.home-assistant.io/integrations/co2signal",
```

Reply:
I figured we could change the name here and add a redirect in https://github.com/home-assistant/home-assistant.io/pull/28116/files so that the URL is less confusing. Is that not possible? Otherwise I'll add the suggested change :)

Reply:
I have applied to suggestion to avoid it being blocking, then we can deal with this at a later stage.